test_scripts/API/SDL_Passenger_Mode/commonPassengerMode.lua

```diff
@@ -0,0 +1,108 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+
+--[[ Module ]]
+local c = actions
+
+--[[ Variables ]]
+
+c.OnDDValue = { "DD_ON", "DD_OFF" }
+c.value = { true, false }
+
+--[[ Common Functions ]]
+
+--[[ @ptuFunc: Update PT
+--! @parameters:
+--! pTbl - table for update
+--! @return: none
+--]]
+function c.ptuFunc(pTbl)
+  pTbl.policy_table.functional_groupings["Base-4"].rpcs.OnDriverDistraction.hmi_levels = { "FULL", "LIMITED", "BACKGROUND" }
+end
+
+--[[ @registerApp: register mobile application
+--! @parameters: none
+--! @return: none
+--]]
+function c.registerApp()
+  c.getMobileSession():StartService(7)
```

Comment:
@HSavynetska There is no need to copy/paste the whole internals of the function from `actions` module. Please just invoke `registerAppWOPTU()` function and add additional expectation on `OnDriverDistraction` notification